tfhe/src/integer/gpu/list_compression/server_keys.rs

```diff
@@ -207,16 +220,20 @@ impl CudaCompressionKey {
             streams.synchronize();
         };
 
-        CudaPackedGlweCiphertextList {
-            data: packed_glwe_list,
+        let meta = Some(CudaPackedGlweCiphertextListMeta {
             glwe_dimension: compressed_glwe_size.to_glwe_dimension(),
             polynomial_size: compressed_polynomial_size,
             message_modulus,
             carry_modulus,
             ciphertext_modulus,
-            bodies_count: num_lwes,
             storage_log_modulus: self.storage_log_modulus,
             lwe_per_glwe: LweCiphertextCount(compressed_polynomial_size.0),
+        });
+
+        CudaPackedGlweCiphertextList {
+            data: packed_glwe_list,
+            meta,
+            bodies_count: num_lwes,
```

Comment:
Maybe the bodies_count could be in the meta as well?

Reply:
It would look better if this was in the meta as well I think, I don't know if we want a len() method as this is the total number of LWEs in the list, I think keeping it like this would be enough.